data/trie/trieStorageManager.go

```diff
@@ -142,6 +143,7 @@ func getSnapshotsAndSnapshotId(snapshotDbCfg config.DBConfig) ([]data.SnapshotDb
 			DBWriteCacher: db,
 		}
 
+		log.Debug("restored snapshot", "snapshot ID", snapshotName)
```

Comment:
log.Trace?

Reply:
Not needed. This will print just a few lines, and just when the node starts/restarts.